git stdiff --git a/tests/algorithms/scaling/test_scale.py b/tests/algorithms/scaling/test_scale.py
index 1a90af14a..bc9646f28 100644
--- a/tests/algorithms/scaling/test_scale.py
+++ b/tests/algorithms/scaling/test_scale.py
@@ -537,7 +537,8 @@ def test_scale_normal_equations_failure(dials_data, tmp_path):


 @pytest.mark.xfail(
-    sys.platform == "darwin" and platform.machine() == "arm64",
+    sys.platform == "win32"
+    or (sys.platform == "darwin" and platform.machine() == "arm64"),
     reason="CC1/2 somewhat differs for unknown reasons",
 )
 def test_scale_and_filter_image_group_mode(dials_data, tmp_path):
diff --git a/tests/command_line/test_export.py b/tests/command_line/test_export.py
index 3b345433b..c60b34e48 100644
--- a/tests/command_line/test_export.py
+++ b/tests/command_line/test_export.py
@@ -2,6 +2,7 @@

 import filecmp
 import json
+import sys

 import procrunner
 import pytest
@@ -564,6 +565,10 @@ def test_export_sum_or_profile_only(dials_data, tmp_path):
         assert (tmp_path / f"removed_{remove}.mtz").is_file()


+@pytest.mark.xfail(
+    sys.platform == "win32",
+    reason="Comparison to reference fails on windows, because of universal newlines",
+)
 @pytest.mark.parametrize("intensity_choice", ["profile", "sum"])
 def test_pets(dials_data, tmp_path, intensity_choice):
     expt = dials_data("quartz_processed", pathlib=True) / "integrated.expt"
